.github/workflows/contracts-near.yml

```diff
@@ -64,5 +64,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Execute
+        env:
+          API_KEY: ${{ secrets.INFURA_API_KEY }} 
```

Comment:
The name should be more descriptive than just `API_KEY`. At least, it should be an `INFURA_API_KEY`

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/818/commits/171b5d3add39392abbc0c543e407b855c68dd86b